analysis/stripe/analysis.sql

```diff
@@ -39,7 +31,7 @@ churns AS (
     sum(total) as total
   from invoices
   where last_payment = 1
-  and date_trunc('month', date) < date_trunc('month', date_trunc('month', current_date) - 1)
+  and date_trunc('month', date) < date_trunc('month', current_date)
```

Comment:
@jthandy I thought this was a bit unwieldy as well, but the new version does something different. 

It looks like the old version checks for a gap of more than one month. Example: last invoice may have been in February. If we are in March right now, the client has not necessarily churned, but the new version will say that they did.

Probably better to just use

date_trunc('month', date) < date_trunc('month', dateadd('month', -1, current_date))